faker/database_structure_manager.py

```diff
@@ -39,6 +60,14 @@ def create_database_tables_structure(database_connection: sqlite3.Connection) ->
 
 
 def check_data_existence(path_to_database: str) -> bool:
+    # Configure logging to output to stdout immediately
+    logging.basicConfig(
+        stream=sys.stdout,
+        level=logging.DEBUG,
+        format="%(asctime)s - %(levelname)s - %(message)s",
+        force=True,  # Override any existing configurations
+    )
+
```

Comment:
You don't need to store this in function. You can define logger config under the import sections.

I recommend watching this YT video to better understand logging. 
https://youtu.be/9L77QExPmI0?si=gVYOoMAzEevNCBzS